corehq/sql_proxy_accessors/sql_templates/create_case_notify_trigger.sql

```diff
@@ -0,0 +1,9 @@
+CREATE OR REPLACE FUNCTION case_notify() RETURNS trigger AS $$
+DECLARE
+BEGIN
+  PERFORM pg_notify('{{ channel.name }}', row_to_json(NEW));
```

Comment:
this is kind of cool. with triggers you have access to the `NEW` row and the `OLD` one. and postgres has a function to turn a row to json